modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_data_set.scss

```diff
@@ -39,3 +39,15 @@
 		}
 	}
 }
+
+.data-set-wrapper {
+	border: 2px solid transparent;
+}
+
+.data-set-wrapper.drop-target {
+	& *:not(.btn-primary):not(.lexicon-icon-plus):not(.lexicon-icon-plus use) {
```

Comment:
I'm keeping the + button out of the highlight style changes. Sample does not show this